docs/api/features/placement.md

```diff
@@ -1,3 +1,137 @@
 # Placement
 
-What are the default rules in SDC, do they differ in hosted Triton?
+When using Docker with a Triton data center (a.k.a. SDC) as your Docker Host,
+your Docker containers can be placed on any physical server in the data center.
+This is one of the basic benefits of Triton's Docker solution: with just the
+vanilla `docker` client you can get the benefit of spreading your containers
+across multiple servers, setting the groundwork for high availability (HA)
+services and avoiding single points of failure (SPOF).
+
+This document explains how Triton places new containers on servers and what
+facilities are exposed for controlling placement.
+
+
+## Default placement
+
+By default, Triton attempts to spread all containers (and non-Docker containers
+and VMs) owned by a single account across separate physical servers.
```

Comment:
@askfongjojo From my recent updates to the DAPI algorithm here (https://github.com/joyent/sdc-designation/blob/master/lib/algorithms/soft-filter-locality-hints.js#L63):

```
 * - default: If no "locality" is given, this algoritm does a non-strict
 *   attempt to filter out servers with VMs owned by this customer. In other
 *   words, by default Triton attempts to spread out a customer's VMs.
 *   Customer-facing documentation should promise no more than Triton making
 *   a reasonable attempt to spread out customer VMs by default -- i.e.
 *   exactly how should be left under-specified to allow for future changes.
 *   (Dev Note: Future changes might be: better balancing, balancing by
 *   RBACv2 project, balancing by image, limiting the number of CNs a
 *   customer can land on. This balancing by default *overrides* subsequent
 *   sorting/scoring attempts in the sdc-designation pipeline.)
```

This is why I used "attempts" in the language above. But perhaps that still gives the wrong impression to a user that may have lots of containers. I'll try to give a more accurate expectation there.

I also wouldn't mind making a quick attempt at an improvement here: to have the default account-spread in the locality algorithm not totally give up if the customer has one VM on every server. A decent attempt at that might be best to wait for the "score"ing stuff that Marsell is working on in DAPI-298/DAPI-301. E.g. we'd bias slightly in favour of CNs without a lot of the account's VMs. Come RBACv2 we'll probably want to do it by project anyway.